ui/css/treeherder-navbar.css

```diff
@@ -140,9 +140,21 @@ secondary-nav-bar {
     width: 100%;
 }
 
+.watched-repo-menu {
+  min-width: 10rem;
+  word-wrap: break-word;
+  max-width: 250px;
+}
+
+.watched-repo-menu > .dropdown-header {
```

Comment:
Can you give me some context on why these CSS items are needed?  With a long message, it really DOES seem like we need what you have here for ``.watched-repo-menu > .dropdown-header``.  But the others don't really seem necessary.  But I might be missing their reasoning.  :)  Do you think we can do without them?

Reply:
I can drop the extra word-wrap without a problem. The min/max-widths are to try to standardize the size of the dropdowns. Otherwise, repos with MotDs or closure reasons have menus that are much wider than repos without those messages. 

Happy to drop it all if that's fine with you.

Reply:
If it ends up needing it, that's totally fine.  :)  I've just recently become aware of how bloated our ``.CSS`` is.  @sarah-clements had brought it up to me that we can get some good mileage out of https://getbootstrap.com/docs/4.1/utilities/sizing/ and the other utilities in there.  If you wouldn't mind, would you see if there's anything in there that can be used instead?  If not, no worries, do what you need to.  :)